### ISSUE TITLE
[MDEP-308] maven-dependency-plugin:2.2:get just download single jar file even if transitive property is set to true

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-dependency-plugin#755]
**[Amit Jolly](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=jolly_amit)** opened **[MDEP-308](https://issues.apache.org/jira/browse/MDEP-308?redirect=false)** and commented

I am trying to see if can download maven artifact transitively (without pom file) from maven repository (Nexus in my case) to local server.

As my POC I am trying to download one of the existing artifact cxf-rt-frontend-jaxws from my local nexus repo.

Below is the command i am using to download. Now the issue is it only downloads the single jar(cxf-rt-frontend-jaxws-2.2.9.jar) file and does not download any of the dependencies.. ( I am using latest version of mvn 3.0.3)

But same artifact if i add in the pom file of one of my java/maven projects, then it does downloads all the required artifacts.

$ mvn -c org.apache.maven.plugins:maven-dependency-plugin:2.2:get -Dtransitive=true -DrepoUrl=http://loneqidelappu3.uk.db.com:8081/nexus/content/groups/common -Dartifact=org.apache.cxf:cxf-rt-frontend-jaxws:2.2.9
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building Maven Stub Project (No POM) 1
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- maven-dependency-plugin:2.2:get (default-cli) @ standalone-pom ---
Downloading: http://loneqidelappu3.uk.db.com:8081/nexus/content/groups/common/org/apache/cxf/cxf-rt-frontend-jaxws/2.2.9/cxf-rt-frontend-jaxws-2.2.9.jar
Downloaded: http://loneqidelappu3.uk.db.com:8081/nexus/content/groups/common/org/apache/cxf/cxf-rt-frontend-jaxws/2.2.9/cxf-rt-frontend-jaxws-2.2.9.jar (302 KB
at 253.6 KB/sec)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 2.328s
[INFO] Finished at: Sat Mar 26 23:49:04 EDT 2011
[INFO] Final Memory: 3M/7M
[INFO] --------------------------------------------------------------------

---

**Affects:** 2.2

**Issue Links:**
- [MDEP-284](https://issues.apache.org/jira/browse/MDEP-284) Get Mojo ignores the transitive attribute because of a typo in the parameter declaration

1 votes, 1 watchers

**Comment by jira-importer** [comment]
**[Mike Deck](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=mikedeck)** commented

I believe this is actually a symptom of the bug described by MDEP-284.  From what I can tell there is a typo on line 147 of GetMojo.java which is causing the value of the `transitive` field in that class to always be false.

I downloaded the latest source for the maven-dependency-plugin from trunk and did nothing more than change {$transitive} to ${transitive} as the description from MDEP-284 suggests.  I ran an install and then executed the dependency:get goal with the newly installed 2.3-SNAPSHOT version of the plugin and it downloaded everything transitively as expected.
